### ISSUE TITLE
Bug mineur d'animation: comportement étrange d'un arbre

### ISSUE THREAD

**Issue by Salvador1312** [JulienNigon/SpellcasterUniversity-Bug-Report#810]
Malheureusement c'était en pleine action et j'ai pas pensé à prendre le screenshot, mais j'ai vu cet arbre (en forêt) avoir un comportement étrange dans le mouvement de ses branches: elles étaient toutes tirées vers le bas à droite. Bon c'est juste un détail cosmétique assez rare, et venant d'une forêt magique à la limite faut pas s'étonner mais je le mets là quand même au cas où.

![20200209140408_1](https://user-images.githubusercontent.com/60800668/74335552-c2ffc380-4d9c-11ea-906c-e3e4ddb81df0.jpg)

**Comment by kaori01** [comment]
idem, je n'ai pas pu prendre de screen c'était trop fugace, mais au passage de certains personnages, les arbres s'étirent avant de redevenir normaux ;)

**Comment by Salvador1312** [comment]
Ah cool merci de confirmer, j'avais peur de délirer ;)

**Comment by Neelan22** [comment]
Oui, cet arbre a une volonté propre visiblement... On regardera ça de plus près un peu plus tard ;)

**Comment by Florane88** [comment]
Il m'est arrivé la même chose mais pendant plusieurs secondes sur la carte du désert, lors d'une attaque de chevaliers.
